torch/onnx/experimental/fonnx/v1/export_options.py

```diff
@@ -0,0 +1,12 @@
+@dataclasses.dataclass
+class ExportOptions:
+    """Class for keeping track of ONNX exporter flags can perform."""
+
+    opset_version: Optional[int] = 17
+    """opset_version: Target ONNX opset version"""
+
+    dynamic_shape: Optional[bool]
```

Comment:
`dynamic_shape: Optional[bool]`

Reply:
"""Hints shape information for input/output tensors.
    When True, a best effort is done for setting all inputs as dynamic.
    Otherwise, they are static"""

Reply:
```python
dynamic_shape: bool = True

"""Hints shape information for input/output tensors.
When True, all inputs shapes are considered dynamic.
Otherwise, all inputs shapes are considered static.
"""
```